pkg/endpoint/metrics.go

```diff
@@ -72,45 +73,23 @@ func (s *regenerationStatistics) SendMetrics() {
 // GetMap returns a map which key is the stat name and the value is the stat
 func (s *regenerationStatistics) GetMap() map[string]*spanstat.SpanStat {
 	result := map[string]*spanstat.SpanStat{
-		"waitingForLock":         &s.waitingForLock,
-		"waitingForCTClean":      &s.waitingForCTClean,
-		"policyCalculation":      &s.policyCalculation,
-		"proxyConfiguration":     &s.proxyConfiguration,
-		"proxyPolicyCalculation": &s.proxyPolicyCalculation,
-		"proxyWaitForAck":        &s.proxyWaitForAck,
-		"mapSync":                &s.mapSync,
-		"prepareBuild":           &s.prepareBuild,
-		"total":                  &s.totalTime,
+		"waitingForLock":             &s.waitingForLock,
+		"waitingForPolicyRepository": &s.waitingForPolicyRepository,
+		"waitingForCTClean":          &s.waitingForCTClean,
+		"policyCalculation":          &s.policyCalculation,
+		"proxyConfiguration":         &s.proxyConfiguration,
+		"proxyPolicyCalculation":     &s.proxyPolicyCalculation,
+		"proxyWaitForAck":            &s.proxyWaitForAck,
+		"mapSync":                    &s.mapSync,
+		"prepareBuild":               &s.prepareBuild,
+		"total":                      &s.totalTime,
 	}
 	for k, v := range s.datapathRealization.GetMap() {
 		result[k] = v
```

Comment:
Tho not currently the case, there is a potential here to override an existing stats key during the map merge. 
Not sure if we may want to flag it?

Reply:
Actually it turns out that when the map keys are initialized like this, Go compiler will fail to compile the code if there's a dup map key literal. I tried this by copying the "waitingForLock" key here and got:

```
duplicate key "waitingForLock" in map literal
```